samples/sp_whoIsActive/src/sql/who_is_active_v11_30.sql

```diff
@@ -0,0 +1,5273 @@
+SET QUOTED_IDENTIFIER ON;
```

Comment:
Please ensure we have registered this in the OSS tool. Will we be shipping this extension? If so, please ensure we have a thirdPartyNotices file that includes the sp_whoisactive license as a reference.

Reply:
Will discuss offline whether we will ship this extension or only provide as sample

---
In reply to: [175923723](https://github.com/Microsoft/sqlopsstudio/pull/943#discussion_r175923723) [](ancestors = 175923723)